docs/pages/fund-apps/calculate-fees.md

```diff
@@ -0,0 +1,111 @@
+# Calculate XMTP fees
+
+Use this guide to understand how to calculate your app or agent's estimated XMTP fees.
+
+## Fee types
+
+Each message sent through XMTP can incur these fee types:
+
+- **Messaging fees**: Charged for messages sent through the XMTP Network.
+    - **Base fee**: A flat per-message fee.
+    - **Storage fee**: A fee charged per-byte-day of storage required. A 100-byte message stored for 30 days would be calculated as 3,000-byte-days.
+    - **Congestion fee**: A dynamic fee computed by looking at the recent activity of an originator. Added only during periods of high network activity.
+- **Gas fees**: Charged for messages that require transactions on the XMTP Appchain. Typically, these are messages for group membership, identity, and funder-related updates.
+
+The dollar value of the flat per-message base fee, per-byte-day storage fee, and congestion fee are stored in a smart contract and remain constant for a specified period of time. These fees are set and adjusted through protocol governance.
+
+## Estimate XMTP fees
+
+Use these examples to help you estimate fees for your own app or agent.
+
+### Estimate monthly fees
+
+XMTP uses pay-as-you-go pricing that benefits from network scale. For example:
+
+| Monthly active users | Estimated monthly fees | Fees per user |
+| --- | --- | --- |
+| 10,000 | ~$150/month | $0.015 |
+| 100,000 | ~$1,500/month | $0.015 |
+| 1 Million | ~$15,000/month | $0.015 |
+| 10 Million | ~$150,000/month | $0.015 |
+
+*Based on average usage of 300 messages per user per month*
+
+### Estimate detailed fee breakdowns by message type
+
+| Content type | Typical size | Messages per $1 | Monthly fees (1M messages) |
+| --- | --- | --- | --- |
+| Text message | 1 KB | 20,000 | $50 |
+| Reaction/receipt | 500 bytes | 40,000 | $25 |
+| Photo (compressed) | 500 KB | 40 | $25,000 |
+| Group update | 2 KB | 10,000 | $100 |
+
+### Estimate congestion fees
```

Comment:
@jhaaaa I don't think apps can / should estimate congestion fees.
- If the network load balancing is working as intended, apps will pay effectively 0 in congestion fees. Because congestion fees are calculated at the node level, they are basically just a flag that says "go use another node that doesn't have congestion fees!"
- It's difficult to make informed assumptions about the inputs to this algorithm.

IMO it's enough to convey the role of congestion fees at a high level and give developers a place to go to learn more about how this works technically if they are curious.